kratos/utilities/read_materials_utility.cpp

```diff
@@ -245,32 +286,177 @@ void ReadMaterialsUtility::AssignPropertyBlock(Parameters Data)
             const auto& r_input_var = KratosComponents<Variable<double>>().Get(input_var_name);
             const auto& r_output_var = KratosComponents<Variable<double>>().Get(output_var_name);
 
-            CheckIfOverwritingTable(*p_prop, r_input_var, r_output_var);
+            if (Data.Has("model_part_name")) {
```

Comment:
IDem

Reply:
It is because the subproperties has not SubModelpart assigned (necessarily), so the check is done prior. This does not affect current implementtaion

Reply:
I would pass as a parameter to the function a "treat_as_subproperty" flag or something, so that it becomes obvious why it is changed (and here check that instead)